server/files/entrypoint_nginx.sh

```diff
@@ -104,6 +104,7 @@ sync_files(){
     for DIR in $(ls /var/www/MISP/app/files.dist); do
         rsync -azh --delete "/var/www/MISP/app/files.dist/$DIR" "/var/www/MISP/app/files/"
     done
+    cp /var/www/MISP/app/webroot/img/orgs.dist/* /var/www/MISP/app/webroot/img/orgs
```

Comment:
I'm debating if this should be rsync similar to above. This might take me a bit to think about.